src/components/appointment/translation.ts

```diff
@@ -50,5 +51,20 @@ const appointmentMessages = {
       defaultMessage: '※此紀錄不會公開給學員看到',
     },
   }),
+
+  AppointmentCancelModal: defineMessages({
+    confirmCancelAlert: {
+      id: 'appointmentMessages.AppointmentCancelModal.confirmCancelAlert',
+      defaultMessage: '確定要取消預約嗎？',
+    },
+    confirmCancelNotation: {
+      id: 'appointmentMessages.AppointmentCancelModal.confirmCancelNotation',
+      defaultMessage: '取消預約後將會寄送通知給諮詢老師，並重新開放此時段，若需退費請主動聯繫平台。',
+    },
+    canceledReason: {
+      id: 'appointmentMessages.AppointmentCancelModal.canceledReason',
```

Comment:
NITS: translation's id should be appointment.{fileName}.key

<img width="701" alt="image" src="https://user-images.githubusercontent.com/28040359/173728023-c1e54c45-6325-4a19-983a-f930024b1bc9.png">